cylc/flow/task_remote_mgr.py

```diff
@@ -501,7 +501,7 @@ def file_install(self, platform):
             self.bad_hosts -= set(platform['hosts'])
             self.ready = True
         else:
-            log_platform_event('file install', platform, host)
+            log_platform_event('starting remote file install', platform, host)
```

Comment:
For the other events we only log when they start (if they fail we get warnings/errors):

```suggestion
            log_platform_event('remote file install', platform, host)
```